crawl-ref/source/items.cc

```diff
@@ -2843,12 +2843,21 @@ static bool _is_option_autopickup(const item_def &item, bool ignore_force)
     return Options.autopickups[item.base_type];
 }
 
+static int _get_chunk_count() {
+    auto it = find_if(you.inv.begin(), you.inv.end(), [&](const item_def& item) {
```

Comment:
Should just be `[]` here, not `[&]`, since there are no locals we need to capture. Makes no difference to the compiler, of course, but looks a bit weird to a human.